hw/ip/tlul/data/tlul_testplan.hjson

```diff
@@ -2,21 +2,21 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
-  // Use the name 'xbar' for this generic testplan. The actual testplan that imports
+  // Use the name 'main' for this generic testplan. The actual testplan that imports
   // this file can override the name if needed.
-  name: xbar
+  name: main
```

Comment:
nit: IMO `name = {xbar_main, xbar_peri, xbar_chip}` feels better - `name` should represent the DUT being verified. I suggest calling this `variant` instead, if you want to stick to this convention.

Reply:
only `name` can be substituted 
Try following code
```
  name: main
  xbar_name: more
  entries: [
    {
      name: xbar_sanity
      desc: '''Sequentially test each host to access any device'''
      milestone: V1
      tests: ["xbar_{name}_{xbar_name}_sanity"]
    }
```
it gave below output
`|     V1      |         xbar_sanity          | xbar_chip__sanity                     |     0     |    0    |    -- %     |`

Reply:
Will revisit this later.